src/backend/src/services/change-requests.services.ts

```diff
@@ -130,6 +139,13 @@ export default class ChangeRequestsService {
             detail: buildChangeDetail('Duration', String(foundCR.wbsElement.workPackage.duration), String(updatedDuration))
           }
         ];
+
+        // update all the dependencies (and nested dependencies) of this work package so that their start dates reflect the new duration
+        if (foundPs.timelineImpact > 0) {
```

Comment:
could be negative too right? or do we not want that to update it

Reply:
Matt says safer is to only delay not push forward, but Casey might disagree.